src/client/components/DisasterQuestion/index.js

```diff
@@ -41,13 +41,19 @@ function DisasterQuestion(props) {
   return (
     <Form.Group controlId="disaster-question">
       <Form.Label>{props.questionText}</Form.Label>
-      <Form.Control as="select" onChange={onChange} value={choice}>
-        {disasterDropDownChoices.map((choice) => (
-          <option key={choice} value={choice}>
-            {choice}
-          </option>
+      <div id="disaster-list">
```

Comment:
Do you mind adding validation to this field as well? You can base it off of this PR: https://github.com/cagov/unemployment/pull/359

Reply:
Added validation

Already confirmed with Tina via Slack with a screenshot and she was good with it.